src/plugins/migration-notice.js

```diff
@@ -0,0 +1,13 @@
+import { SET_REFERRED } from '~/store-modules/mutation-types.js'
+
+/**
+ * If the URL contains a referral parameter pointing containing
+ * 'creativecommons.org' the migration notice needs to be displayed. This
+ * plugin checks the presence of the param and updates the store.
+ */
+export default function ({ query, store }) {
+  if ('referrer' in query && query.referrer) {
```

Comment:
Is the double check necessary? Could it just be condensed to just check `query.referrer`?

Reply:
You're right, 3f0b28c should fix this.